spec/requests/api/members_spec.rb

```diff
@@ -2,10 +2,6 @@
 
 describe "api/members" do
 
-  def json
```

Comment:
it's only used in one place in the file and helpers shouldn't really be defined in the spec file itself